samples/CustomPolicyProvider/Startup.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CustomPolicyProvider
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // An IAuthorizationPolicyProvider can take the place of the usual AddAuthorization call
```

Comment:
Note: this isn't really true, the reason you don't have to call AddAuthorization is because AddMvc does it...  So I would probably replace this comment with something like "This replaces the default policy provider..."

Reply:
Ah, I didn't realize that. I've fixed the comment.